File: 07_salary_by_hours/python/main.py
if __name__ == '__main__':
    print("\n--- PROGRAMA PARA CALCULAR SALARIO POR HORAS SEMANALES ---\n")

    hours = int(input("Ingrese las horas trabajadas durante la semana > "))

    if hours <= 48:
        salary = hours * 15000
    elif hours <= 70:
        salary = 48 * 15000 + (hours - 48) * 18000
    else:
        salary = 48 * 15000 + (hours - 48) * 22000

    print(f"\nTu salario para esta semana es de: $ {salary}")

File: 07_salary_by_hours/rust/src/main.rs
use std::io;

fn main() {
    let mut input = String::new();
    println!("\n--- PROGRAMA PARA CALCULAR SALARIO POR HORAS SEMANALES ---\n");

    println!("Ingrese las horas trabajadas durante la semana:");
    io::stdin().read_line(&mut input).unwrap();
    let hours = input.trim().parse::<u32>().expect("Las horas deben ser un número entero positivo");

    let salary = if hours <= 48 {
        hours * 15000
    } else if hours <= 70  {
        48 * 15000 + (hours - 48) * 18000
    } else {
        48 * 15000 + (hours - 48) * 22000
    };

    println!("\nTu salario para esta semana es de: ${salary}");
}
